azure/pas-full-network.json

```diff
@@ -67,28 +67,25 @@
     }
   },
   "variables": {
-    "virtualNetworks_Vault_Vnet_name": "Vault-Vnet",
-    "subnets_Vault_Subnet_name": "Vault-Subnet",
-    "networkSecurityGroups_NSG_Vault_name": "NSG-Vault",
-    "virtualNetworks_Vault_DR_Vnet_name": "Vault-DR-Vnet",
-    "subnets_Vault_DR_Subnet_name": "Vault-DR-Subnet",
-    "virtualNetworks_Components_Vnet_name": "Components-Vnet",
-    "subnets_Components_Subnet_name": "Components-Subnet",
-    "networkSecurityGroups_NSG_Components_name": "NSG-Components",
-    "subnets_Web_Subnet_name": "Web-Subnet",
-    "networkSecurityGroups_NSG_Web_name": "NSG-Web",
-    "virtualNetworkPeerings_Vault_Components_Peering_name": "Vault-Components-Peering",
-    "virtualNetworkPeerings_Components_Vault_Peering_name": "Components-Vault-Peering",
-    "virtualNetworkPeerings_Vault_VaultDR_Peering_name": "Vault-VaultDR-Peering",
-    "virtualNetworkPeerings_VaultDR_Vault_Peering_name": "VaultDR-Vault-Peering",
-    "virtualNetworkPeerings_VaultDR_Components_Peering_name": "VaultDR-Components-Peering",
-    "virtualNetworkPeerings_Components_VaultDR_Peering_name": "Components-VaultDR-Peering"
+    "virtualNetworks_Solution_VNet_name": "PAS-VNet",
```

Comment:
Why subnets are beginning with the component name, and NSG beginning with NSG?
Please be consistent?

Reply:
Done.